render/render.go

```diff
@@ -81,6 +82,12 @@ func render(configFiles []composetypes.ConfigFile, finalEnv map[string]string) (
 	if err := processEnabled(rendered); err != nil {
 		return nil, err
 	}
+	for ix, service := range rendered.Services {
```

Comment:
I think this feature should be tested (overriding the service images name using the `imageMap`).